containers/pre_built_workers/node/build_qps_worker.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright 2020 gRPC authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+npm install -production
```

Comment:
nit: Let's use 2 dashes for long-form flags and 1 dash for short-form flags. For example, `-v` and `--verbose`.

```suggestion
npm install --production
```

Reply:
Thanks for reviewing:) Done.